src/fairly/__init__.py

```diff
@@ -117,8 +114,30 @@ def get_repository(id: str) -> Dict:
             return repository
     return None
 
+def write_default_config(repositories: list) -> None:
+    """
+    Write the default configuration file
+    >>> fairly.write_default_config()
+    """
+    # For each client generate a default configuration
+    config = {}
+    # Add repositories list to config
+    config["repositories"] = []
+    for id, repository in repositories.items():
+        platform = { "platform": repository['client_id'], "token": "" }
+        exists = platform in config["repositories"]
+        if not exists:
+            config["repositories"].append(platform)
+
+    # Write the default configuration
+    with open(os.path.expanduser("~/.fairly/config.json"), "w") as file:
+        json.dump(config, file, indent=4)
```

Comment:
The existing implementation intentionally not creating a default configuration, because a) it pollutes home directory of the user if custom configuration is not necessary (e.g. use of the package to clone public datasets), b) default user configuration by definition should be empty, as we have a centralized configuration file of the known repositories under the package folder. Information available in the centralized configuration should not be repeated in the user configuration. I suggest we don't create a default configuration. We can describe how to create custom configuration in detail in the documentation.